### ISSUE TITLE
The cover photo for the Containers lesson should have a proper alt text

### ISSUE THREAD

**Issue by iampava** [FrontEnd-ro/frontend.ro#322]
Right now the alt text is `TODO` 😅

**Comment by iampava** [comment]
Done!
